tasks/apply-mapping/apply-mapping.yaml

```diff
@@ -87,7 +87,7 @@ spec:
                 variable=$(jq -c --argjson i "$i" '.[$i]' <<< "${SUPPORTED_VARIABLES}")
                 KEY=$(jq -r 'to_entries[] | .key' <<< $variable)
                 VALUE=$(jq -r 'to_entries[] | .value' <<< $variable)
-                tags=$(echo -n $tags | sed "s/$KEY/$VALUE/g")
+                tags=$(echo -n $tags | sed "s|$KEY|$VALUE|g")
```

Comment:
Now what happens if it has `|` in it? ;) Why would there be `/` in tags? Is that supported? I don't think it should even be allowed for tags.

Reply:
If the snapshot is malformatted, the digest tag can come back with a `/` in it. The pipeline then fails, even if you don't want to use the digest tag. I don't see any situation where someone has a `|` in their tags, so I think this is an improvement (not a huge one, but one)

Reply:
As discussed in chat, this doesn't hurt anything, so I'm fine with this. But we should probably add a check to fail if the containerImage reference is invalid (i.e. not using sha reference).